loadero_python/resources/run.py

```diff
@@ -1,11 +1,11 @@
 """Loadero run resource.
 
-Run resources is seperated into three parts:
-    - RunParams class that describes runs attributes
+Run resources is seperated into three parts
+    RunParams class that describes runs attributes
 
-    - RunAPI class implements run resources API operations
+    RunAPI class implements run resources API operations
 
-    - Runs class that in combination with RunParams and RunAPI allows to manage
+    Runs class that in combination with RunParams and RunAPI allows to manage
     a single run resources instance.
```

Comment:
for other classes it seems there are no newlines inbetween these

Reply:
overall there doesn't seem to be consistency for these comments - in some cases they are indented, in others not. in some cases they have dashes preceeding each item